package.json

```diff
@@ -25,7 +25,7 @@
     "apollo-utilities": "^1.3.4",
     "@sentry/browser": "6.3.1",
     "@sentry/integrations": "6.3.1",
-    "@smontero/ppp-client-api": "0.0.43",
```

Comment:
@arsenijesavic Please use `@smontero/ppp-client-api` for now. I need to test and ensure that `@hypha-dao/ppp-client-api` is working correctly